src/App.js

```diff
@@ -1,27 +1,17 @@
-import './styles/App.css';
-import { HashRouter as Router, Switch, Route } from "react-router-dom";
-import { Home, Menu, Restaurants } from "./pages";
+import React, { Component } from 'react';
+import './App.css';
+import Form from './components/Form';
+import 'bootstrap/dist/css/bootstrap.min.css';
 
-function App() {
-  return (
-    <div className="App">
-      <Router>
-        <Switch>
-          <Route exact path="/" component={Home} />
-          <Route path="/menu" component={Menu} />
-          <Route path="/restaurants" component={Restaurants} />
-          {/* Setup dymanic routing for restaurant pages  */}
-          {/* <Route path="/restaurants">
-            <Route path="/antarctic" component={Restaurants} />
-            <Route path="/arctic" component={Restaurants} />
-            <Route path="/sahara" component={Restaurants} />
-            <Route path="/arabian" component={Restaurants} />
-            <Route path="/gobi" component={Restaurants} />
-          </Route>*/}
-        </Switch> 
-      </Router>
-    </div>
-  );
+class App extends Component {
+  render() {
+    return (
+        <div className="App">
+         <Form />
+        </div>
+    )
```

Comment:
We do not want to change the App.js These changes should be removed from this pull request. Form component can be added to the restaurant pages in a future pull request.